app/javascript/css/main.scss

```diff
@@ -0,0 +1,251 @@
+
+html, body {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  padding: 0;
+}
+
+body {
+  color: #333;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+}
+
+.screen {
+  width: 100%;
+  height: 100%;
+}
+
+a {
+  color: black;
+  text-decoration: none;
+
+  &:hover {
+    text-decoration: none;
+  }
+
+  &:visited {
+    color: black;
+  }
+}
+
+label {
+  display: block;
+}
+
+input {
+  font-family: inherit;
+  font-size: inherit;
+  -webkit-padding: 0.4em 0;
+  margin: 0 0 0.5em 0;
+  border: 1px solid #ccc;
+  box-sizing: border-box;
+  border-radius: 2px;
+}
+button {
+  font-family: inherit;
+  font-size: inherit;
+  -webkit-padding: 0.4em 0;
+  margin: 0 0 0.5em 0;
+  border: 1px solid #ccc;
+  box-sizing: border-box;
+  border-radius: 2px;
+}
+select {
+  font-family: inherit;
+  font-size: inherit;
+  -webkit-padding: 0.4em 0;
+  margin: 0 0 0.5em 0;
+  border: 1px solid #ccc;
+  box-sizing: border-box;
+  border-radius: 2px;
+}
+textarea {
+  font-family: inherit;
+  font-size: inherit;
+  -webkit-padding: 0.4em 0;
+  margin: 0 0 0.5em 0;
+  border: 1px solid #ccc;
+  box-sizing: border-box;
+  border-radius: 2px;
+}
+
+input {
+  &:disabled {
+    color: #ccc;
+  }
+}
+
+button {
+  color: #333;
+  background-color: #f4f4f4;
+  outline: none;
+
+  &:disabled {
+    color: #999;
+  }
+
+  &:not(:disabled):active {
+    background-color: #ddd;
+  }
+
+  &:focus {
+    border-color: #666;
+  }
+}
+
+#for_form {
+  display: flex;
+  transition: opacity .5s;
+  float: left;
+  justify-content: center;
+  flex-wrap: wrap;
+  background: rgb(255, 255, 255);
+  width: 350px;
+  min-height: 250px;
+  color: #333;
+  margin-top: 50px;
+  box-shadow: 0 6px 8px rgba(0, 0, 0, 0.5);
+}
+
+.button_login {
+  text-decoration: none;
+  display: inline-block;
+  color: white;
+  padding: 20px 30px;
+  margin: 10px 20px;
+  border-radius: 10px;
+  font-family: 'Montserrat', sans-serif;
+  text-transform: uppercase;
+  letter-spacing: 2px;
+  background-image: linear-gradient(to right, #9EEFE1 0%, #201d3d 51%, #9EEFE1 100%);
+  background-size: 200% auto;
+  box-shadow: 0 0 20px rgba(0, 0, 0, .1);
+  transition: .5s;
+  &:hover {
+    background-position: right center;
+  }
+}
+
+.for_button {
+  display: flex;
+  width: 400px;
+  height: 80px;
+}
+
+.center {
+  display: flex;
+  justify-content: center;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+#for_form {
```

Comment:
you have already styles for that id, move into one place
in general - avoid styling for #id, use classes only, id's are used for js